src/plugins/persistence/couch/pluginSpec.js

```diff
@@ -41,13 +41,12 @@ describe('the plugin', () => {
                 namespace: '',
                 key: 'some-value'
             },
-            type: 'mock-type',
+            type: 'notebook',
```

Comment:
changing type to `notebook` so observer gets called, which was the source of the bug.